dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskConstants.java

```diff
@@ -187,6 +188,25 @@ private TaskConstants() {
      */
     public static final String PARAMETER_TASK_INSTANCE_ID = "system.task.instance.id";
 
+    /**
+     * the definition name of current task
+     */
+    public static final String PARAMETER_TASK_DEFINITION_NAME = "system.task.definition.name";
+
+    /**
+     * the instance id of the process to which current task belongs
+     */
+    public static final String PARAMETER_PROCESS_INSTANCE_ID = "system.process.instance.id";
```

Comment:
how about use `workflow` as keyword instead of `process`? because our web ui use workflow.